boards/arm/lpcxpresso55s36/lpcxpresso55s36.dts

```diff
@@ -143,20 +143,33 @@
 	pinctrl-names = "default";
 };
 
+/* Flash is divided into 32 kB sub-regions.
+ * Each sub-region can be assigned individual
+ * security tier in secure AHB controller.
```

Comment:
just curious, do you have an opinion on how a zephyr user would interact with the AHB controller?

Reply:
> just curious, do you have an opinion on how a zephyr user would interact with the AHB controller?

There is no any option. MCUx SDK does not have AHB driver.